include/splinterdb/splinterdb.h

```diff
@@ -321,4 +321,18 @@ splinterdb_stats_print_lookup(const splinterdb *kvs);
 void
 splinterdb_stats_reset(splinterdb *kvs);
 
+// Set the streams where informational and error messages will be printed.
+//
+// These default to stdout and stderr, respectively.
+//
+// A program may use this function to override the defaults, to send
+// the messages to a file, or /dev/null or elsewhere.
+//
+// Typical usage would be to silence info messages, but leave error
+// messages going to stderr:
+//   splinterdb_log_streams(fopen("/dev/null", "w"), stderr);
+void
+splinterdb_log_streams(platform_log_handle *info_stream,
```

Comment:
(As you remarked elsewhere) I prefer the name to be `splinterdb_set_log_streams()`. The term `_set_` makes it clear as to what's going on. (Also will take away the potential ambiguity of `_log_streams` implying we are `logging'` something. `log` is also a verb.)

Reply:
Add-on comment: Btw, I see this comment elsewhere in `platform.c`:

```
platform_linux/platform.c:16:// platform_error_log() messages go to stderr. These can be changed to module-
platform_linux/platform.c-17-// or test-specific log files, by overriding these settings.
platform_linux/platform.c-18-platform_log_handle *Platform_default_log_handle = NULL;
```

It will be useful to sneak-in a reference to your new method by updating L18 to something like:

```
or test-specific log files, by using the splinterdb_set_log_streams() interface.
```

Reply:
On the name: I've changed it to `splinterdb_logs_set_streams`.  This includes the `_set_` that you asked for, but retains using `_logs_` as a sort of sub-namespacing.  That is intended to follow the pattern that Alex and I discussed recently here: https://github.com/vmware/splinterdb/pull/459#discussion_r1000921275.  That way, any future functions related to message logs can have the same prefix.

Reply:
Good catch on the code comment in `platform.c`, I've updated it to mention the new function.